dlpy/tests/test_tensorboard.py

```diff
@@ -0,0 +1,482 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# NOTE: This test requires a running CAS server.  You must use an ~/.authinfo
+#       file to specify your username and password.  The CAS host and port must
+#       be specified using the CASHOST and CASPORT environment variables.
+#       A specific protocol ('cas', 'http', 'https', or 'auto') can be set using
+#       the CASPROTOCOL environment variable.
+
+import os
+import shutil
+import swat
+import swat.utils.testing as tm
+from swat.cas.table import CASTable
+from swat.cas.results import CASResults
+from dlpy.model import Model, Optimizer
+from dlpy.sequential import Sequential
+from dlpy.tensorboard import TensorBoard
```

Comment:
please get these into individual tests (actually for this class you can put in the setup as they are all tensorboard tests). and skip if tf / tb not installed